### PR TITLE
events page added

### PR DESCRIPTION
**Pull request by Mahadevinc**

### CODE REVIEW

File: src/components/events.js [hunk 1]
```diff
@@ -1,11 +1,90 @@
 import React, {Component} from 'react';
+import { Jumbotron } from 'react-bootstrap';
```
Comment (akramkazmi71) [inline src/components/events.js:2]: Add Row, Col, Carousel in this same import statement. No need to add a separate import for each Attribute.
Lets keep all the imports from bootstrap in a single place.

File: src/components/events.js [hunk 2]
```diff
@@ -1,11 +1,90 @@
...
+                                    <Col sm={7} className="d-flex justify-content-start">
+                                        <div className="d-flex align-items-center">
+                                            <p className="eventhead">
+                                            ISE TECH CLUB EVENTS
+                                            </p>
+                                        </div>
+                                    </Col> 
+                                    <Col sm={5} className="d-flex justify-content-center">  
+                                        <div className="eventImage">
+                                            <img src={ Events } alt=" Events " className="imgfluid" />
```
Comment (akramkazmi71) [inline src/components/events.js:31]: Remove this in mobile view, it does not look good in mobile view. You can refer the notes page for this.

File: src/components/events.js [hunk 3]
```diff
@@ -1,11 +1,90 @@
...
+            <div>
+
+                <Jumbotron className="eventsHeader">
+                    <section>
+                        <Container>
+                                <Row>
+                                    <Col sm={7} className="d-flex justify-content-start">
+                                        <div className="d-flex align-items-center">
+                                            <p className="eventhead">
+                                            ISE TECH CLUB EVENTS
```
Comment (akramkazmi71) [inline src/components/events.js:25]: Let make this banner the same size as Notes banner. So it will look good.

**Review by akramkazmi71 — CHANGES_REQUESTED** [review]

**Comment by akramkazmi71** [comment]
Merging this page for now, but can you try making this page using a json file?
Just like what we did with notes and team page.

**Comment by Mahadevinc** [comment]
> Merging this page for now, but can you try making this page using a json file?
> Just like what we did with notes and team page.

sure. I will try to plan that with json file

**Pull request merged**